### DISCUSSION TITLE
Local Proxy 模式下Grpc客户端如何实现负载均衡

### DISCUSSION THREAD

**Discussion by dominicqi** [apache/rocketmq#7483] [Q&A]
各位大佬，咨询个问题
Local模式下每个Proxy和内部的Broker进行通信，那么客户端连接到Proxy的时候如何进行负载均衡呢？是不同的实例配置不同的proxy地址吗？

**Comment by dominicqi** [comment]
看懂了，客户端负载均衡默认使用的是pick first, 所有proxy都会返回全量的 topicRoute，每个Queue的broker信息在Local模式下就是固定的broker地址

**Reply by huxiyu** [reply]
请问下，proxy外能加层负载均衡LB吗？比如nginx或f5，因为只想暴露LB的一个ip(域名)和一个端口到公网。
我用Pulsar的proxy已经实现，不知道RokcetMQ 5.x可以否？
